packages/@okta/vuepress-site/.vuepress/public/img/icons/icon--menu-white.svg

```diff
@@ -0,0 +1 @@
+<svg width="19" height="15" fill="none" xmlns="http://www.w3.org/2000/svg"><path stroke="#fff" stroke-width="1.75" stroke-linecap="round" stroke-linejoin="round" d="M1.078 1.034h16.551M1.078 7.579h16.551M1.078 14.125h16.551"/></svg>
```

Comment:
I think it's OK either way probably, but could we avoid loading copies of the same SVG by setting the stroke color via CSS

Reply:
Pretty sure you can only do this with inline SVGs

Reply:
Oh, it's used as a background image. 😞